src/callbacks/stepsize.jl

```diff
@@ -0,0 +1,76 @@
+
+"""
+    StepsizeCallback(; cfl=1.0)
+
+Set the time step size according to a CFL condition with CFL number `cfl`
+if the time integration method isn't adaptive itself.
+"""
```

Comment:
I used this also I used the same convention also for other callbacks such as `SaveSolutionCallback`. The idea behind that is to document what's defined in the file at the top to give readers of the code an impression of what's going on. I basically document the constructor since that's what people will use in elixirs.